app/src/main/java/com/microsoft/device/samples/dualscreenexperience/presentation/catalog/CatalogListViewModel.kt

```diff
@@ -7,6 +7,7 @@
 
 package com.microsoft.device.samples.dualscreenexperience.presentation.catalog
```

Comment:
I think you forgot to add the OSS license header.

Reply:
It's an old file and since the OSS license was not modified, you have to click the "expand" button from Github to see it:)